llvm/include/llvm/Target/GlobalISel/Combine.td

```diff
@@ -1560,6 +1560,13 @@ def combine_shuffle_concat : GICombineRule<
         [{ return Helper.matchCombineShuffleConcat(*${root}, ${matchinfo}); }]),
   (apply [{ Helper.applyCombineShuffleConcat(*${root}, ${matchinfo}); }])>;
 
+// Combines shuffles of vector into extract_subvector
```

Comment:
This isn't producing an extract_subvector?

Reply:
ugh, forgot to update it.